# CropLine.msg

float32 m1
float32 b1

float32 m2
float32 b2